batched/KokkosBatched_Util.hpp

```diff
@@ -626,7 +626,6 @@ KOKKOS_INLINE_FUNCTION auto subview_wrapper(ViewType v, IdxType1 i1,
                                             const Trans::NoTranspose) {
   return subview_wrapper(v, i1, i2, i3, layout_tag);
 }
-#if KOKKOS_VERSION < 40099
 template <class ViewType, class IdxType1>
```

Comment:
Isn't this the branch that should be deleted (the one for old versions)?

Reply:
Sure however it looks like both branch are identical, probably pointing at the fact that the "old code" was probably not compiling correctly? My suspicion is that `Kokkos::ALL_t` was supposed to be something like `Kokkos::Impl::ALL_t`?

Reply:
Oh, makes sense :)